src/streamsync/core.py

```diff
@@ -204,6 +203,9 @@ def ingest(self, raw_state: Dict) -> None:
         for key, raw_value in raw_state.items():
             self.__setitem__(key, raw_value)
 
+    def items(self) -> Sequence[Tuple[str, Any]]:
```

Comment:
What is this? Is this necessary or a separate improvement to make it look more like a Dict?